resources/zone/gb_ci_vat.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "United Kingdom - Channel Islands (VAT)",
+    "scope": "tax",
+    "members": [
+        {
+            "type": "country",
+            "id": "gb_ci_vat_0",
+            "name": "Jersey",
+            "country_code": "JE",
+            "included_postal_codes": "\/(^$)|(^JE[0-9]{1})\/"
+        },
+        {
+            "type": "country",
+            "id": "gb_ci_vat_1",
+            "name": "Jersey",
+            "country_code": "GB",
+            "included_postal_codes": "\/(^JE[0-9]{1})\/"
```

Comment:
Why do we need two zone members for Jersey, with a very similar regexp? Can't we just say "JE + a number, anywhere in the string"?